subprojects/gradle/namespaced-resources-fixer/src/test/kotlin/com/avito/android/plugin/FileResourcesFixerTest.kt

```diff
@@ -0,0 +1,147 @@
+package com.avito.android.plugin
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import java.io.File
+
+class FileResourcesFixerTest {
```

Comment:
I expect to see how Fixer will work with aliases in imports e.g.
`com.avito.android.R as app_R`

Reply:
Added new test `replace library id - simple property with alias to R`